ground/src/main/java/com/google/android/ground/ui/map/gms/GoogleMapsFragment.kt

```diff
@@ -149,11 +143,6 @@ class GoogleMapsFragment : SupportMapFragment(), MapFragment {
     watermark.layoutParams = params
   }
 
-  override fun onCreate(savedInstanceState: Bundle?) {
-    super.onCreate(savedInstanceState)
-    customCap = CustomCap(bitmapUtil.bitmapDescriptorFromVector(R.drawable.ic_endpoint))
```

Comment:
Is the removal of the custom drawable here meant to be in this PR? Is this part of the revamp work for the Polygon UI?

Reply:
This code is unused at the moment and was also preventing the unit test from running. While working on Polygon UI, I'll have to figure out how to implement it correctly.